CRM/Contribute/Form/ContributeFormTrait.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+use Civi\API\EntityLookupTrait;
+
+/**
+ * Trait implements functions to retrieve contribution related values.
+ */
+trait CRM_Contribute_Form_ContributeFormTrait {
+
+  use EntityLookupTrait;
+
+  /**
+   * Get the value for a field relating to the event.
```

Comment:
again @eileenmcnaughton this doc block seems to be incorrect for this trait as well